wolfcrypt/src/port/cavium/cavium_octeon_sync.c

```diff
@@ -0,0 +1,995 @@
+/* cavium_octeon.c
+ *
+ * Copyright(C) 2006-2019 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.(formerly known as CyaSSL)
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
```

Comment:
Check License version.

Reply:
Fixed, and updated the filename at the top of the file.